luxtronik/parameters.py

```diff
@@ -893,9 +893,9 @@ def __init__(self, safe=True):
             849: Unknown("ID_Ba_Hz_MK3_saved"),
             850: Hours("ID_Einst_Kuhl_Zeit_Ein_akt", True),
             851: Hours("ID_Einst_Kuhl_Zeit_Aus_akt", True),
-            852: Unknown("ID_Waermemenge_Seit"),
+            852: Energy("ID_Waermemenge_Seit"),
             853: Unknown("ID_Waermemenge_WQ"),
-            854: Unknown("ID_Waermemenge_Hz"),
+            854: Energy("ID_Waermemenge_Hz"),
```

Comment:
`Hz` probably means Heizung in this context?

Reply:
Yes, I think so.